package.json

```diff
@@ -27,6 +27,7 @@
     "lint-md": "remark -q *.md src/**/*.md configs/*.md docs/**/*.md",
     "lint-fix": "yarn lint-js -- --fix",
     "test": "jest --forceExit",
+    "test-coverage": "yarn test -- --coverage",
```

Comment:
I wasn't sure if we wanted add coverage to the default test script instead of a creating a new one.